time_tracker_api/users/users_namespace.py

```diff
@@ -68,7 +69,14 @@ def get(self):
         """List all users"""
         user_role_field_toggle = FeatureToggleManager('bk-user-role-field')
         if user_role_field_toggle.is_toggle_enabled_for_user():
-            return AzureConnection().users_v2()
+            azure_connection = AzureConnection()
+            current_user_is_tester = azure_connection.is_test_user(
```

Comment:
maybe this could be a better name `is_current_user_a_tester`

Reply:
Done.